packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -254,18 +255,24 @@ function errorMessageFromResponse(
     : '';
 }
 
+interface DestinationCertificateJson {
+  Type: string;
+  Name: string;
+  Content: string;
+}
```

Comment:
[nth] This is the same type as the `DestinationCertificate` - I believe it could be closer to each other so that the difference is obvious.
[nth] Also, because the types are almost the same, check this out: https://www.typescriptlang.org/docs/handbook/2/mapped-types.html#key-remapping-via-as.